dnf/module/repo_module_dict.py

```diff
@@ -331,24 +331,15 @@ def get_info_profiles(self, module_spec):
         subj = ModuleSubject(module_spec)
         module_version, module_form = subj.find_module_version(self)
 
-        table = smartcols.Table()
-        table.noheadings = True
-        table.column_separator = " : "
-
-        column_name = table.new_column("Name")
-        column_value = table.new_column("Value")
-        column_value.wrap = True
-
-        line = table.new_line()
-        line[column_name] = "Name"
-        line[column_value] = module_version.full_version
+        lines = OrderedDict()
+        lines["Name"] = module_version.full_version
 
         for profile in module_version.profiles:
-            line = table.new_line()
-            line[column_name] = profile
-            line[column_value] = " ".join(module_version.profile_nevra(profile))
+            nevra_objects = module_version.profile_nevra_objects(profile)
```

Comment:
Please what is here nevra? Is it Nevra hawkey class?

Reply:
yes